### PR TITLE
Atomic Hosting: Rename section to Hosting Configuration

### LINKED ISSUE #37584: Hosting: Add header, and subheader to page, Change Section Name
The fix the flows team are working on implementing a heading pattern across dotcom (link to [Calypso doc](https://wpcalypso.wordpress.com/devdocs/design/formatted-header)) 

Ongoing work in https://github.com/Automattic/wp-calypso/pull/37343

Also, let's rename the section name to "Hosting Configuration", since "Hosting Access" and "SFTP & MySQL" doesn't fit as well with planned future features. The route name should also match the new name -> `/hosting-config`. See p1574259922359400-slack-atomic

**Mock of page with header/subheader:**
Screen Shot 2019-11-14 at 7 40 25 PM

### LINKED ISSUE #37585: Hosting: make left column cards 70%, right column 30%
**Current:**
Screen Shot 2019-11-13 at 7 35 36 PM

**Should be:**
Screen Shot 2019-11-14 at 7 06 04 PM

### PR DESCRIPTION
**Pull request by mmtr**
#### Changes proposed in this Pull Request

* Renames the section to "Hosting Configuration".
* Changes the route to `/hosting-config/:site`.
* Applies some style adjustments to the layout so it matches the latest mockups. 

#### Screenshots

Atomic site with SFTP user created

| Viewport | Before | After |
|---|---|---
| Desktop |  |  |
| Tablet |  |  |
| Mobile |  | |

Atomic site without SFTP user created

| Viewport | Before | After |
|---|---|---
| Desktop |  | Screen Shot 2019-11-21 at 11 37 36 |
| Tablet |  | Screen Shot 2019-11-21 at 11 37 55 |
| Mobile |  | Screen Shot 2019-11-21 at 11 38 18 |

Business site no transferred to Atomic

| Viewport | Before | After |
|---|---|---
| Desktop |  | Screen Shot 2019-11-21 at 11 42 00 |
| Tablet | | Screen Shot 2019-11-21 at 11 42 21 |
| Mobile |  | Screen Shot 2019-11-21 at 11 42 38 |

#### Testing instructions

* Make sure the hosting section is displayed according to the screenshots above.
* Make sure the hosting section is named as "Hosting Configuration" (sidebar, browser tab name) and it is accessed through the `/hosting-config/:site` route.

Fixes #37585
Fixes #37584

### CODE REVIEW

File: client/my-sites/hosting/data-loss-warning/index.js [hunk 1]
```diff
@@ -14,30 +14,25 @@ import './style.scss';
...
+			<CardHeading>{ translate( 'Support' ) }</CardHeading>
+			<div className="data-loss-warning__avatars">
 				{ shuffle( avatars ).map( avatar => (
 					<Gravatar key={ avatar } user={ { avatar_URL: avatar } } size={ 42 } />
 				) ) }
 			</div>
-		</div>
+			<p>
+				{ translate(
+					'If you need help or have any questions, our Happiness Engineers are here when you need them.'
```
Comment (a8ci18n) [inline client/my-sites/hosting/data-loss-warning/index.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'If you need help or have any questions, our Happiness Engineers are here when you need them!' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/atomic-hosting-layout)

File: client/my-sites/hosting/support-card/index.js [hunk 2]
```diff
@@ -14,33 +14,28 @@ import './style.scss';
...
+			<CardHeading>{ translate( 'Support' ) }</CardHeading>
+			<div className="support-card__avatars">
 				{ shuffle( avatars ).map( avatar => (
 					<Gravatar key={ avatar } user={ { avatar_URL: avatar } } size={ 42 } />
 				) ) }
 			</div>
-		</div>
+			<p>
+				{ translate(
+					'If you need help or have any questions, our Happiness Engineers are here when you need them.'
```
Comment (a8ci18n) [inline client/my-sites/hosting/support-card/index.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'If you need help or have any questions, our Happiness Engineers are here when you need them!' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/atomic-hosting-layout)

File: client/my-sites/hosting/phpmyadmin-card/index.js [hunk 3]
```diff
@@ -52,54 +57,50 @@ const PhpMyAdminCard = ( { translate, siteId, token, loading, disabled } ) => {
...
-				<MaterialIcon icon="dns" size={ 32 } />
-			</div>
-			<div>
-				<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
-				<p>
+			<MaterialIcon icon="dns" size={ 32 } />
+			<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
+			<p>
+				{ translate(
+					'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
```
Comment (a8ci18n) [inline client/my-sites/hosting/phpmyadmin-card/index.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Manage your databases with phpMyAdmin and run a wide range of operations with MySQL.' )` *ES Score: 10*

**Comment by mmtr** [comment]
@olaolusoga I decided to show the SFTP labels/values in independent lines on small screens so it's more readable. Let me know if this should be different:

File: client/my-sites/hosting/support-card/index.js (hunk 2 above)
Comment (mmtr) [reply client/my-sites/hosting/support-card/index.js]: We dropped the exclamation in the latest mockup.

File: client/my-sites/hosting/phpmyadmin-card/index.js (hunk 3 above)
Comment (mmtr) [reply client/my-sites/hosting/phpmyadmin-card/index.js]: There is only one database, so we've adjusted the copy to better reflect that.

**Comment by olaolusoga** [comment]
Moving values to second line looks good

**Pull request merged**